newIDE/app/src/Profile/AdditionalUserInfoDialog.js

```diff
@@ -158,6 +162,15 @@ const AdditionalUserInfoDialog = ({
     disableUnderline: true,
   };
 
+  if (currentlyRunningInAppTutorial) {
+    // TODO: Do not display dialog to not distract/force user to provide more info
```

Comment:
Maybe also set the prop `open` of the dialog to `!currentlyRunningInAppTutorial` so that this dialog is not even opened (to prevent a possible flash in the UI)